src/components/dialog/content/error/ReportIssueButton.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <Button
```

Comment:
We probably need a mechanism to tell user who they are sending to? A tooltip might be helpful here. For example: `Report the error to Comfy-Org`.